pages/introduction/privacy-policy/README.md

```diff
@@ -0,0 +1,32 @@
+---
+navigation_title: "Privacy Policy"
+position: 5
+changed: "2020-04-30"
+---
+
+# Privacy Policy
+
+In order for us to provide you the best possible experience on our guide, we need to collect and process certain information.
+
+## Usage data
+
+When you visit our site, we will store amongst other things:
+
+- The website from which you visited us from
+- The parts of our site you visit
+- The date and duration of your visit
+- Your anonymised IP address
+- Information from the device (device type, operating system, screen resolution, language, country you are located in, and web browser type) you used during your visit
+
+We process this usage data in [Matomo Analytics](https://matomo.org/) for statistical purposes, to improve our site.
+
+## Cookies
+
+We use cookies (small data files transferred onto computers or devices by sites) for record-keeping purposes and to enhance functionality on our site. You may deactiva
+
+## Opt-out of website tracking
+
+<iframe
+        style="border: 0; height: 300px; width: 100%;"
+        src="https://adg-analytics.nothing.ch/index.php?module=CoreAdminHome&action=optOut&language=de&fontColor=272727&fontSize=20px&fontFamily=sans-serif"
```

Comment:
First of all, we need to display English text.

In addition, the whole element looks a bit out of place design-wise:

![image](https://user-images.githubusercontent.com/1814983/85555602-5dfb8a80-b626-11ea-8d1e-8453950e5b2a.png)

Instead of including an iframe, wouldn't it be possible to place the code directly into the `README.md` file? AFAIK, markdown allows the usage of HTML, although maybe some elements are forbidden for security purposes?

Reply:
e.g. We would like to improve the usability of this site for you and other users. In order to do this, we need some analytic data. You can choose that your actions on our website are not tracked. This will protect your privacy, but it will prevent us from learning from your visit, your actions, etc. Currently, you are being tracked by Matomo Web Analytics. Uncheck this box to opt-out.

Reply:
> 
> 
> Last but not least: is it enough to place this information and opt-out option deeply hidden in the website? Shouldn't it be advertised on the homepage or something?

I would rather have a pop-up or overlay, similar to the cookies message you get everywhere.

Reply:
> 
> 
> I added a link to the policy to the footer. Better than nothing - but enough??

The link goes to licence instead of Privacy policy. And the page should probably be "Privacy policy" with only one capital P.

Reply:
> I would rather have a pop-up or overlay, similar to the cookies message you get everywhere.

As agreed upon in our call, we feel that providing a link in the footer is enough.